js/graph_config.js

```diff
@@ -226,7 +229,9 @@ GraphConfig.load = function(config) {
             }
         }
 
-        var getMinMaxForFields = function(/* fieldName1, fieldName2, ... */) {
+/*
+//!!!!!! The stats data have small issues of min-max data !!!! 
```

Comment:
Usually we mark these with a TODO
```suggestion
TODO - The stats data have small issues of min-max data !!!! 
```